cypress/page/products-list.page.ts

```diff
@@ -3,8 +3,8 @@ class ProductsListPage {
   private checkoutBtn: string;
 
   constructor() {
-    this.addCarBtn = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-    this.checkoutBtn = "[style*='display: block;'] .button-container > a";
+    this.addCarBtn = '#center_column a.button.ajax_add_to_cart_button.btn.btn-default';
+    this.checkoutBtn = '.button-container > a[title="Proceed to checkout"]';
```

Comment:
probably just `a[title="Proceed to checkout"]` should work, or maybe a contains with "Proceed to checkout"

Reply:
resolved